search-app/src/cmr/search/services/acls/collections_cache.clj

```diff
@@ -116,62 +116,35 @@
     (doseq [[coll-key coll-value] collections-map]
       (hash-cache/set-value cache cache-key coll-key (clj-times->time-strs coll-value)))))
 
-;; TODO this is the new func that will replace 'get-collection' AND 'get-collection-map' during transition
-(defn get-collection-gran-acls
- "Gets a single collection from the cache by concept id. Handles refreshing the cache if it is not found in it.
- Also allows provider-id and entry-title to be used."
+(defn get-collection-for-gran-acls
  ([context coll-concept-id]
+  "Gets a single collection from the cache by concept id. If collection is not found, will add it to the cache (if it exists in elastic) and will return the found collection."
   (let [coll-by-concept-id-cache (hash-cache/context->cache context coll-for-gran-acl-caches/coll-by-concept-id-cache-key)
+        _ (info (str "coll cache for concept id = " (pr-str coll-by-concept-id-cache)))
```

Comment:
I just want to ask, are these info logs needed or could they be debug. Don't actually answer me here, just consider it.

Reply:
Its nice to be able to query splunk to see when a cache was refreshed.  I have needed this a couple of times.